src/utils/self_test/SConscript

```diff
@@ -6,27 +6,43 @@
 """Build self test"""
 
 import os
+import re
 import daos_build
+Import('env', 'prereqs', 'cart_lib', 'gurt_lib')
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(_str):
+    """function to remove gcc flag for self_test compilation """
+    r = re.compile(r"-Wframe-larger-than=")
```

Comment:
(pylint-invalid-name) Variable name "r" doesn't conform to snake_case naming style